app/models/order_item.rb

```diff
@@ -5,9 +5,4 @@ class OrderItem < ActiveRecord::Base
 
   validates :order_id, presence: true
   validates :item_id, presence: true
-  validates :quantity, presence: true
-
-  def total_dollar_amount
-    number_to_currency(line_item_price / 100.00)
-  end
 end
```

Comment:
since we woudn't have multiple quantities of a ticket, no longer need for order_item price since it'll be the same as the item.unit_price